depmapomics/config.py

```diff
@@ -89,7 +121,7 @@
 rnasource7 = "ibm"
 
 ## our working workspace (reference)
-rnaworkspace = "broad-firecloud-ccle/DepMap_hg38_RNAseq"
+RNAWORKSPACE = "broad-firecloud-ccle/DepMap_hg38_RNAseq"
 
 ## curent WGS GP buckets
 wgsworkspace1 = "terra-broad-cancer-prod/DepMap_WGS"
```

Comment:
maybe capitalize this as well?

Reply:
I could, I wanted to leave them uncapitalize to really see the distinction with the compute workspace / gp data workspaces.